hw/drivers/trng/trng_kinetis/syscfg.yml

```diff
@@ -30,3 +30,10 @@ syscfg.defs:
         value: 0
         restrictions:
             - '!KINETIS_TRNG_USE_RNGA'
+    KINETIS_TRNG_ENTROPY_DELAY:
+        description: >
+            The entropy delay can be used to improve the speed of number
+            generation with the compromise of worse entropy. 3200 is the
+            default value, but it can be reduced for faster value
+            generation.
```

Comment:
I have not read documentation of TRNG block. Maybe description could specify what the units are.

Reply:
I rephrased it including the "units".